other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
```

Comment:
Define "foreground", I don't understand what you mean by that in this context. Daemons never run in foreground, they are started by some init system.